packages/fcl/src/current-user/exec-service/strategies/ext-rpc.js

```diff
@@ -0,0 +1,67 @@
+import {invariant} from "@onflow/util-invariant"
+import {extension} from "./utils/extension"
+import {normalizePollingResponse} from "../../normalize/polling-response"
+import {configLens} from "../../../config-utils"
+
+export function execExtRPC(service, body, opts) {
```

Comment:
What is you destructured service here `execExtRPC({ endpoint, params, data, type }, body, opts)` and then later in the send you could just do `service: { params, data, type }` and also `extInstalled(endpoint)`?

Reply:
makes sense, wasn't sure yet if we wouldn't need any other data from the service